.gitattributes

```diff
@@ -1,2 +1 @@
-secrets/** filter=git-crypt diff=git-crypt
```

Comment:
Why has this been removed? Have the secrets been migrated elsewhere?

Reply:
This is an old remnant for when git-crypt was used in the binderhub repo to encrypt the SSH key for helm chart publishing or similar.